apps/nowcasting-app/components/charts/remix-line.tsx

```diff
@@ -483,6 +483,8 @@ const RemixLine: React.FC<RemixLineProps> = ({
                         typeof value[1] !== "number"
                       )
                         return null;
+                      if ((key === "PROBABILISTIC_RANGE" && value[0] < 0) || value[1] < 0)
```

Comment:
`if ((key === "PROBABILISTIC_RANGE" && (value[0] < 0) || value[1] < 0))`

 `A and (B or C)` rather than `A and B or C` which might not work

Reply:
if its less than 0, should we just return 0.0, you can image -0.00000001 we do not want to be NULL but perhaos 0

Reply:
yeah, I think that's a good idea. I also think it might be better to change the code that formats the data for the chart instead of the tooltip. I'll have a look at this again Monday.